scripts/crypto.getPrice.js

```diff
@@ -10,5 +10,5 @@ try {
   const response = await axios.get(url, { params });
   return response.data[currency].usd;
 } catch (error) {
-  throw new Error(`Could not fetch price for ${currency}: ${error.message}`);
+  throw new Error("Could not fetch price for " + currency + ": " + error.message);
```

Comment:
these scripts have been modified with this syntax because the previous syntax wasn't working with handlebars, we could add a helper if you'd like but I found this change non-harmful - let me know your thoughts @dOrgJelli

Reply:
left a comment regarding this, should fix the root of the problem